### PR TITLE
PolarGrid: Add angular grid

### PR DESCRIPTION
**Pull request by patrickelectric**
Signed-off-by: Patrick José Pereira <patrickelectric@gmail.com>

### CODE REVIEW

File: qml/PolarGrid.qml [hunk 1]
```diff
@@ -103,4 +103,29 @@ Item {
             }
         }
     }
+
+    Canvas {
+        anchors.fill: parent
+        opacity: 0.8
+        property var angle: 45
+
+        onPaint: {
```
Comment (jaxxzer) [inline qml/PolarGrid.qml]: Does this redraw every time we paint a new profile? If so, is that necessary? Can it just belong on another layer/item?

Comment (patrickelectric) [reply qml/PolarGrid.qml]: No. What is necessary ? Why it should be in another item ? This is the polar grid, it should deal with grid stuff.

Comment (jaxxzer) [reply qml/PolarGrid.qml]: My question is:

Does **this item** get drawn/painted once at initialization, or repeatedly every time the visualizer updates the sensor data?

Comment (jaxxzer) [reply qml/PolarGrid.qml]: (when is onPaint called)

Comment (patrickelectric) [reply qml/PolarGrid.qml]: As said before. No.
Paint is only called when the region needs to be rendered, like resize events.

**Pull request merged**